.changes/next-release/feature-e7c0a41d-61ad-479f-aac3-fac8f1c42dd5.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I dont think we need the changelog, users shouldn't care about this change

Reply:
makes sense, will remove